docs/_docs/contributing.md

```diff
@@ -8,7 +8,7 @@ Hi there! Interested in contributing to Jekyll? We'd love your help. Jekyll is a
 
 ## Where to get help or report a problem
 
-See [the support guidelines](SUPPORT.md)
+See [the support guidelines](https://github.com/jekyll/jekyll/blob/master/.github/SUPPORT.md)
```

Comment:
Might need to change this in `.github/CONTRIBUTING.md`.

Reply:
OK, I wasn't sure it would work :)

Reply:
If you edit locally then run `bundle exec rake site:generate`.

Reply:
The current link works fine [from GitHub](https://github.com/jekyll/jekyll/blob/master/.github/CONTRIBUTING.markdown)

It's when we generate the [contributing page](https://jekyllrb.com/docs/contributing/) through the `Rakefile`, that the link to `SUPPORT.md` doesn't work anymore.
If we generate a support page for the website, and link with `../support`, it would break the link on GitHub. I don't see any other simple way to make it work on both pages.